android-json-form-wizard/src/main/java/com/vijay/jsonwizard/fragments/JsonFormFragment.java

```diff
@@ -827,6 +827,11 @@ public boolean onMenuItemClick(MenuItem item) {
     @Override
     public void onStop() {
         super.onStop();
+    }
+
+    @Override
+    public void onDestroy(){
+        super.onDestroy();
         presenter.cleanUp();
```

Comment:
We can add a unit test and use Mockito to verify that the _cleanUp_ method of the presenter has been invoked when one calls the _onDestroy_ method of the **JsonFormFragment**

Reply:
@ndegwamartin noted

Reply:
Added the test